pages/agent/v3/signed-pipelines.md

```diff
@@ -0,0 +1,114 @@
+# Signed pipelines
+
+## What are signed pipelines?
+
+Signed Pipelines is a security feature on the Buildkite Agent that allows you to cryptographically sign steps that are uploaded Buildkite, and then verify that signature on the Buildkite Agent before running the step. This can prevent a malicious actor from modifying the steps that are run on your agents, especially in the (hopefully unlikely) event that the Buildkite backend is compromised.
+
+These signatures mean that while a threat actor could modify jobs in flight, when the Agent runs those steps, when it tries to verify the signature, it will fail, and the step will not be run.
+
+<details>
+  <summary>I think I've seen this before...</summary>
+  This work is inspired by the older <a href="https://github.com/buildkite/buildkite-signed-pipeline"><code>buildkite-signed-pipeline</code></a> tool, which was a tool you could add to your agent instances. It had a similar idea - signing steps before they're uploaded to Buildkite, then verifying them when they're run. However, it had a number of limitations, including:
+  <ul>
+    <li>It had to be installed on every agent instance, leading to more configuration</li>
+    <li>It only supported symmetric HS256 signatures, meaning that every verifier could also sign uploads</li>
+    <li>It couldn't sign matrix steps</li>
+  </ul>
+
+  This newer version of pipeline signing is built right into the agent, and addresses all of these limitations. Being built into the agent, it's easier to configure and use.
+
+  Many thanks to Seek.com.au, who we collaborated with on the older version of the tool, and who have been instrumental in the development of this newer version.
+</details>
```

Comment:
This'll need a pass from someone with a better sense than I of the tone we use in our docs